api/config.py

```diff
@@ -147,6 +147,21 @@
 
 assert input_schemas == expected_input_schemas, '{} is different from {}'.format(input_schemas, expected_input_schemas)
 
+def create_or_recreate_ttl_index(coll_name, index_name, ttl):
+    index_list = db[coll_name].index_information()
+    if index_list:
+        for index in index_list:
+            if index_list[index]['key'][0][0] == index_name:
```

Comment:
I could be convinced to just assume indexes will be generated with the name `<index_name>_1` to get rid of this for loop and [0][0] disaster.